### DISCUSSION TITLE
Loading an autosave does not correctly load layers

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#14376] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

title

https://github.com/user-attachments/assets/4301a362-b7f7-4b7e-a2ef-f003a8627164

adding a new layer seems to refresh the list and solve the issue but it's annoying to do every time you load an autosave

### Reproduction steps

1. Make a submarine.
2. Use layers.
3. Open an autosave of this submarine.
4. oops no more layers

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### -

Submarine Editor

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/d7ca9635f3c9aea2873792dc0ad80505f7fa600e (commit in our private repo). Fix coming up in a future update!
